chromium/background.js

```diff
@@ -118,32 +119,40 @@ chrome.webNavigation.onCompleted.addListener(function() {
   updateState();
 });
 
+var USER_RULE_KEY = 'userRules';
+// Records which tabId's are active in the HTTPS Switch Planner (see
+// devtools-panel.js).
+var switchPlannerEnabledFor = {};
+// Detailed information recorded when the HTTPS Switch Planner is active.
+// Structure is:
+//   switchPlannerInfo[tabId]["rw"/"nrw"][resource_host][active_content][url];
+// rw / nrw stand for "rewritten" versus "not rewritten"
+var switchPlannerInfo = {};
```

Comment:
Could this stuff (`USER_RULE_KEY`, `switchPlannerEnabled/Info`) be moved into `initializeStoredGlobals`?

Reply:
`initializeStoredGlobals` is more for those stored via the storage API.